temoa/extensions/get_comm_tech.py

```
@@ -70,9 +71,9 @@ def get_scenario(inp_f):
     return dict(OrderedDict(sorted(scene_list.items(), key=lambda x: x[1])))
 
 
-def get_comm(inp_f, db_dat):
-    comm_list = {}
-    comm_set = set()
+def get_comm(inp_f: str, db_dat: bool) -> OrderedDict[str, str]:
```

Ruff (0.14.10)

Boolean-typed positional argument in function definition

(FBT001)